src/pages/homepage.tsx

```diff
@@ -93,117 +97,235 @@ const HomePage = () => {
           </Column>
         </Container>
       </Container>
-      {/* CTA public */}
-      <Container layout="wide" gray={true} paddingY={false}>
-        <Container gridCols="12" gridGutters="11" paddingX={false} gray={true}>
-          <ButtonBlockList title={t('homepage:cta_public.title')}>
-            <ButtonBlock
-              title={t('homepage:cta_public.case_studies.title')}
-              iconName="FiTwitter"
-              iconColor="var(--colorBlack400)"
-              white={true}
-            >
+
+      {/* Instances */}
+      <Container gray={true} layout="wide" paddingX={false}>
+        <Container gridCols="10" gridGutters="9" paddingTop={false}>
+          <CardList title={t('instances:title')} centerTitle={true} big={true}>
+            {instancesData.instances.map((instance: any) => {
+              const slug = kebabCase(instance.name);
```

Comment:
This is ambiguous with the `slug` of the URLs, and not very clear in the following code.

```suggestion
              const instanceId = kebabCase(instance.name);
```

Reply:
Fixed with https://github.com/ambanum/OpenTermsArchive.org/pull/161/commits/bb493c0b20636a957e5f69f2e2bbc164b8912862